packages/core-commands/package.json

```diff
@@ -34,6 +34,7 @@
 		"@wordpress/core-data": "file:../core-data",
 		"@wordpress/data": "file:../data",
 		"@wordpress/element": "file:../element",
+		"@wordpress/html-entities": "file:../html-entities",
```

Comment:
I think you need to do `npm install` after this change.

Reply:
Done.